### PR TITLE
Panorama byoi

### PR DESCRIPTION
**Pull request by shaniacht1**
fixes #7161

added panorama BYOI
added dynamic block list command
added script dynamic block list
minor changes to release notes test

### CODE REVIEW

File: Integrations/integration-Panorama.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                'file' : file
+              },
+              params.insecure,
+              params.proxy
+            );
+
+            if (res.StatusCode !==200) {
+              throw 'Request Failed.\nStatus code: ' + res.StatusCode +' '+errorDict[res.StatusCode]+ '.\nBody: ' + JSON.stringify(res) + '.';
+            }
+            return JSON.parse(x2j(res.Body));
```
Comment (Spivaka) [inline Integrations/integration-Panorama.yml]: we also had a case where the response was 200, but the xml itself has an error. So we need to check this scenario as well and throw

File: Integrations/integration-Panorama.yml [hunk 2]
```diff
@@ -0,0 +1,153 @@
+versionedfields: {}
+commonfields:
+  id: Panorama
+  version: -1
+name: Panorama
+display: Panorama
```
Comment (Spivaka) [inline Integrations/integration-Panorama.yml]: add from version 3.0.0

Comment (shaniacht1) [inline Integrations/integration-Panorama.yml]: so from now I should write the actual version instead of -1?

Comment (dorsha) [inline Integrations/integration-Panorama.yml]: no.. you need to add:
`fromversion: 3.0.0`

Comment (shaniacht1) [inline Integrations/integration-Panorama.yml]: done

**Pull request merged**